.storybook/config.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import { configure, addDecorator } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@storybook/addon-notes';
+import { withInfo } from '@storybook/addon-info';
+
+import { ThemeProvider } from 'styled-components';
+
+const StyledSelect = styled.select`
+  height: 50px;
+  width: 150px;
+`;
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const SelectionContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const ComponentContainer = styled.div``;
+
+
+
+class AppLayout extends React.Component {
```

Comment:
We should put the layout component into a different file and import it.

Reply:
you are right, I pushed the update